utils/readers.py

```diff
@@ -15,6 +15,7 @@ def read_parameters(param_file):
         YAML (Ruamel) CommentedMap dict-like object
     """
     yaml = YAML()
+    yaml.allow_duplicate_keys=True
```

Comment:
Can you give an example when this is necessary? Wouldn't we want to raise an error if there are duplicate keys?

Reply:
yea,  conf/config_template.yaml has two key instances of "overlap". I've added the bit below. Perhaps it would be easier to relabel them. 

https://github.com/NRCan/geo-deep-learning/blob/6852729d0ec03c0c2a6465b7a68039181c8bcbda/conf/config_template.yaml#L113-L123

Reply:
Thanks for bringing that to our attention. There's shouldn't be duplicate keys here and I'm afraid allowing them would not provide the desired result as, according to [the yaml documentation](https://yaml.readthedocs.io/en/latest/api.html), the second instance of the same key would be discarded. I'll create a "bug" issue I guess so we can correct this ASAP.